widgets/cpu_openbsd.lua

```diff
@@ -0,0 +1,60 @@
+-- cpu_openbsd.lua - provide CPU usage on OpenBSD
```

Comment:
IMHO the filename is redundant (I've checked several GNU projects: they don't seem to include the filenames in the header).  Since there's the same problem with notmuch wtype, I suggest opening a separate PR for it.

Reply:
I'll remove both file headers. The battery module also has that line. I guess I got it from the `spawn.lua` file and thought I had to add it.

Reply:
For examples, please [navigate to the copyright branch](https://github.com/vicious-widgets/vicious/blob/copyright/widgets/bat_openbsd.lua).